crates/holochain_sqlite/Cargo.toml

```diff
@@ -65,6 +65,7 @@ rusqlite = { version = "0.29", features = [
 ] }
 
 [dev-dependencies]
+holochain_sqlite = { version = "0.3.0-beta-dev.8", path = ".", features = ["test_utils", "slow_tests"] }
```

Comment:
This just makes all the tests run with `cargo test` rather than needing to specify these features. Makes it harder to accidentally not run tests